CONTRIBUTING.md

```diff
@@ -420,3 +420,21 @@ partially-converted asynchronous version of the same name to the `test/asynchron
 Use this generated file as a starting point for the completed conversion.
 
 The script is used like so: `python tools/convert_test_to_async.py [test_file.py]`
+
+## Running PyMongo with SSL
```

Comment:
What's the motivation for adding this section?

Reply:
Oh uh, I was struggling to get it working and asked Noah a bunch of questions, so he suggested I add a section here.

Reply:
I'm concerned about people seeing this section and misinterpreting it to mean that they should be using tlsAllowInvalidCertificates. Can we remove it or make it very clear that this is only for local testing (using our test certificates) and not production?